xml/decoderIndex.xml

```diff
@@ -36046,10 +36046,6 @@
         </model>
       </family>
       <family name="IntelliLight" mfg="Uhlenbrock Elektronik" type="stationary" file="Uhlenbrock_IntelliLight2820.xml">
-        <model model="Controller" productID="2800" formFactor="Stationary" comment="Lighting Controller">
-          <versionCV />
-          <size length="600" width="105" height="66" units="mm" />
-        </model>
```

Comment:
This PR looks good, but I'm uncertain about this change since I don't see this in the other files in this PR. I only want to check that this decoder hasn't been removed by misstake.

Reply:
What I mean with my comment is that I don't see the "Lighting Controller" decoder removed in one of the other files in this PR. Maybe it has been removed before and everything is all right, but I don't know.

Reply:
@silverailscolo What's your thoughts about this? Should we merge this PR anyway? I'm fine with that, I just want to check first.

Reply:
Deletion is correct in this case, and - as @bobjacobsen confirmed - this is just the index. Green light.